src/StoreApi/Utilities/OrderController.php

```diff
@@ -274,16 +296,16 @@ protected function validate_email( \WC_Order $order ) {
 	protected function validate_addresses( \WC_Order $order ) {
 		$errors           = new \WP_Error();
 		$needs_shipping   = wc()->cart->needs_shipping();
-		$billing_address  = $order->get_address( 'billing' );
```

Comment:
We should make sure this wasn't in place for a reason.

Reply:
One is filtered. e.g. `woocommerce_get_order_address` so this might be breaking in ways I am unsure of.

Reply:
I'm still not sure I see your point here.

Because we need `$order` in `validate_address_fields`. Previously it was only getting the address, right now, we pass the order and pull from the inside, but in both cases we're using `get_address`

Reply:
My point is `$order->get_address( 'billing' )` is ran through different filters to `$order->get_billing_country()`. It probably doesn't matter, it just struck me as odd we'd use `$order->get_address` here instead of the dedicated function unless there was a reason.